app/views/content-pages/contact.html

```diff
@@ -0,0 +1,23 @@
+<article class="text container">
+
+  <h1>Nous contacter</h1>
+
+  <p>Contactez-nous pour obtenir des informations sur le simulateur Mes&nbsp;Aides, nous signaler une différence de calcul avec un autre service ou bien encore proposer une amélioration. Nous vous répondons directement par courriel ou sur les réseaux sociaux.  Si votre situation est critique, nous avons également listé les solutions d’urgence les plus courantes sur la page <a ui-sref="sos">SOS</a>.</p>
+
+  <p><a analytics-on="click" analytics-event="Support" href="mailto:bonjour@mes-aides.gouv.fr">Nous envoyer un courriel.</a></p>
```

Comment:
Uniformiser avec le _call to action_ en-dessous : passer à l'impératif ou passer l'autre à l'infinitif.